guardian/workflow-frontend
# Upgrade to Angular 1.8

## What does this change?
Following #460, we are now upgrading Angular to latest v1 to fix some high vulnerabilities.

### How to test
Deploy to CODE. Smoke test Workflow. Check the date-time picker works when creating a new article.
